frontend/rsbuild.config.ts

```diff
@@ -4,6 +4,7 @@ import { pluginReact } from '@rsbuild/plugin-react';
 import { pluginSass } from '@rsbuild/plugin-sass';
 import { pluginSvgr } from '@rsbuild/plugin-svgr';
 import { RsdoctorRspackPlugin } from '@rsdoctor/rspack-plugin';
+// import { pluginModuleFederation } from '@module-federation/rsbuild-plugin';
```

Comment:
I think this is the key to actually making the most of the new v2 runtime module federation plugin.

Reply:
https://rsbuild.dev/guide/advanced/module-federation#module-federation-v20